Dockerfile

```diff
@@ -0,0 +1,69 @@
+## Build
+FROM golang:1.22-alpine AS build
```

Comment:
Note: we should also update this on the cli-runner, when comparing saw it was still on `golang:1.21`